src/Bicep.Types/Concrete/ObjectType.cs

```diff
@@ -33,6 +33,10 @@ public enum ObjectPropertyFlags
         WriteOnly = 1 << 2,
 
         DeployTimeConstant = 1 << 3,
+
+        OnlyReturnedOnRead = 1 << 4,
```

Comment:
Have you thought about what this is going to look like when integrated into Bicep? Is the plan to use this information when the concrete object type is constructed, or will we need to carry it around the concrete types themselves in Bicep? The latter could be tricky.

Reply:
These flags are not required for read-only/write-only resource tagging, so I've removed them from this PR and opened #837 for further discussion.